hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/TestQueryTable.scala

```diff
@@ -0,0 +1,302 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.hudi
+
+import org.apache.hudi.HoodieSparkUtils
+import org.apache.hudi.exception.{HoodieDuplicateKeyException, HoodieException}
+import org.apache.spark.sql.functions.lit
+
+class TestQueryTable extends HoodieSparkSqlTestBase {
+
+  test("Test Query None Partitioned Table") {
+    withTempDir { tmp =>
+      val tableName = generateTableName
+      spark.conf.set("hoodie.datasource.v2.read.enable", "true")
```

Comment:
do we support for spark.sql("hoodie.datasource.v2.read.enable", "true")?

Reply:
going to support this.